Domain/Order.java

```diff
@@ -1,3 +1,58 @@
+import java.util.HashMap;
+import java.util.Map;
+
 public class Order {
-    
+    private Integer m_OrderId;
+    private Map<Integer ,ShoppingBasket> m_ShoppingBasketMap; // <ShopId, ShoppingBasketPerShop>
+    private double m_TotalOrderAmount;
+
+    // Constructor
+    public Order(Integer orderId) {
```

Comment:
add comment for later: "//DOTO: add external service to the constructor and the class for payment and shipping methods".